src/jenkins.py

```diff
@@ -114,6 +115,30 @@ def validate(self) -> None:
             ) from exc
 
 
+def _wait_for(func: typing.Callable, timeout: int = 300, check_interval: int = 10) -> None:
```

Comment:
The typing for `func` can be `typing.Callable[[], typing.Any]`

Reply:
Fixed, thank you!